plugins/netscaler-k8s.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: netscaler-k8s
```

Comment:
this is going against our [naming guidelines](https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/#:~:text=Plugin%20names%20should%20not%20include,or%20%E2%80%9Ckubernetes%2D%E2%80%9D%20prefixes.&text=While%20it%20is%20not%20recommended,the%20intended%20use%20is%20clear.) `Avoid repeating kube[rnetes]`. What would be the reason to suffix this name with `-k8s` when it's obvious that it's a kubectl plugin and therefore manages Kubernetes resources?

Reply:
@arijitr-citrix It's ok by me, but I would recommend maybe even going for `netscaler` if you think there wouldn't be any other netscaler plugin for kubectl in the future and you represent the company behind Netscaler here (i.e. you're submitting this branded plugin on the company's behalf).